insights/parsers/yum_list.py

```diff
@@ -5,6 +5,7 @@
 The parsers contains in this module are:
 
 YumListInstalled - Command ``yum list installed``
+YumListAvailable - Command ``yum list available``
```

Comment:
The `----` line is needed for each of the sub-title by the `sphinx-doc`.

See: https://documentation-style-guide-sphinx.readthedocs.io/en/latest/style-guide.html#headings

Reply:
Added.